docs/development/testguide.rst

```diff
@@ -587,15 +582,6 @@ tables to HDF5 format), and in ``test_all`` for dependencies only used in tests
 (e.g., ``skyfield``, which is used to cross-check the accuracy of coordinate
 transforms).
 
-Using pytest helper functions
-=============================
-
-If your tests need to use `pytest helper functions
-<https://docs.pytest.org/en/latest/reference/reference.html#functions>`_, such as
-``pytest.raises``, import ``pytest`` into your test module like so::
-
-    import pytest
```

Comment:
We enforce this with Ruff rule [PT013 (pytest-incorrect-pytest-import)](https://docs.astral.sh/ruff/rules/pytest-incorrect-pytest-import/).

Reply:
Is this rule worth mentioning here, so people know it is actually enforced? Not every contributors are familiar with all aspects of infrastructure.